docker/compose/public_full_node/public_full_node.yaml

```diff
@@ -10,7 +10,7 @@ base:
 execution:
     # Path to a genesis transaction. Note, this must be paired with a waypoint. If you update your
     # waypoint without a corresponding genesis, the file location should be an empty path.
-    genesis_file_location: "/opt/diem/etc/genesis.blob"
+    genesis_file_location: "/tmp/fullnode1/etc/genesis.blob"
```

Comment:
why change this?

Reply:
I think I was testing something- accident :-)